data/src/generate_tsv_summary.py

```diff
@@ -11,15 +11,60 @@
 
 
 def _wiki_name_and_transcription_level(ele: List[str]) -> str:
-    return ele[3] + ele[5]
+    return ele[3] + ele[7]
 
+def _get_modifiers(
+    language: Dict[str, Any], modifier: str
+) -> Dict:
+    try:
+        return language[modifier]
+    except KeyError:
+        return {}
+
+def _handle_script(
+    language: Dict[str, Any], file_path: str
+) -> Dict:
+    scripts = _get_modifiers(language, "script")
+    key = file_path[
+        file_path.index("_") + 1 : file_path.rindex("_phone")
+    ]
+    if "_" in key:
+        key = key[:key.index("_")]
+    if key in scripts:
```

Comment:
This is faster than what you have in lines 33-36.

```
return scripts.get(key, "")
```

Why? It only looks up once (whereas yours looks the key up in 33, and then again in line 34).

Also applies to some of the other _handle functions.